From 9ef3d0fcf57dd3366d190741829c362b7967a449 Mon Sep 17 00:00:00 2001
From: crito <crito@fnordpipe.org>
Date: Thu, 23 Nov 2017 22:34:46 +0100
Subject: [PATCH] fix glibc-2.25 compat issues

sysmacros complete moved to sysmacros.h.
duplicates of types.h are deprecated.
---
 tools/blktap2/control/tap-ctl-allocate.c | 1 +
 tools/libxl/libxl_internal.h             | 1 +
 2 files changed, 2 insertions(+)

diff --git a/tools/blktap2/control/tap-ctl-allocate.c b/tools/blktap2/control/tap-ctl-allocate.c
index 8a6471e987..25f5f6461d 100644
--- a/tools/blktap2/control/tap-ctl-allocate.c
+++ b/tools/blktap2/control/tap-ctl-allocate.c
@@ -34,6 +34,7 @@
 #include <getopt.h>
 #include <libgen.h>
 #include <sys/stat.h>
+#include <sys/sysmacros.h>
 #include <sys/types.h>
 #include <sys/ioctl.h>
 #include <linux/major.h>
diff --git a/tools/libxl/libxl_internal.h b/tools/libxl/libxl_internal.h
index ae16c2525f..6882bd6b73 100644
--- a/tools/libxl/libxl_internal.h
+++ b/tools/libxl/libxl_internal.h
@@ -41,6 +41,7 @@
 #include <sys/poll.h>
 #include <sys/select.h>
 #include <sys/stat.h>
+#include <sys/sysmacros.h>
 #include <sys/time.h>
 #include <sys/types.h>
 #include <sys/wait.h>
-- 
2.15.0

